libs/components/filter-bar/src/lib/modules/filter-bar/filter-item-modal.component.ts

```diff
@@ -34,13 +39,39 @@ import { SkyFilterItem } from './models/filter-item';
   ],
 })
 export class SkyFilterItemModalComponent implements SkyFilterItem {
+  /**
+   * A unique identifier for the filter item.
+   * @required
+   */
   public readonly filterId = input.required<string>();
+
+  /**
+   * The label text for the filter item.
```

Comment:
```suggestion
   * The label to display for the filter item.
```
Very minor edit, so feel free to disregard if it would slow things down. But "label" and "text" back-to-back felt redundant and also repeats the property name. We could alternately just delete "text" and not replace it with "to display."